### ISSUE TITLE
Use dart-sass instead of node-sass?

### ISSUE THREAD

**Issue by mateusbandeiraa** [carbon-design-system/carbon-components-vue#1250]
The project depends on `node-sass`, which is deprecated in favor of using `dart-sass`. I was able to validate that it is possible to compile the project using dart-sass without any source-code change at all, but there were some warnings like:
```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($px, $carbon--base-font-size)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
25 │   @return ($px / $carbon--base-font-size) * 1rem;
   │            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    node_modules\carbon-components\scss\globals\scss\vendor\@carbon\layout\scss\_convert.scss 25:12     carbon--rem()
    node_modules\carbon-components\scss\globals\scss\vendor\@carbon\layout\scss\_breakpoint.scss 16:23  @import
    src\styles\_carbon-utils.scss 2:9                                                                   @import
    src\styles\_carbon.scss 5:9                                                                         @import
    stdin 23:9                                                                                          root stylesheet
```
The fix for these issues is fairly simple.

Would it be a welcome change if I submitted a PR replacing `node-sass` by `dart-sass` and replacing division operations by `math.div`?

**Comment by mateusbandeiraa** [comment]
I actually noticed that the `.scss` files that need to be updated are under carbon-design-system/carbon. We only need to remove `node-sass` and add `dart-sass` to the dev dependencies then.

Actually, this problem was resolved by carbon-design-system/carbon#9508, and we can get this change by updating carbon-components to version 10.43.0 (released some days ago). I'll create a PR addressing this shortly.
